<ui version="4.0" >
 <class>TextProperties</class>
 <widget class="QDialog" name="TextProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MuseScore: Text Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item rowspan="6" row="0" column="0" >
    <widget class="QLabel" name="label_14" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Ignored" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="mscore.qrc" >:/data/bg1.jpg</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="fontSelect" >
       <property name="toolTip" >
        <string>font face</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fontSize" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>1000</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" name="_2" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2_2" >
       <property name="text" >
        <string>Style:</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fontBold" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="toolTip" >
        <string>bold</string>
       </property>
       <property name="text" >
        <string>B</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fontItalic" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="toolTip" >
        <string>italic</string>
       </property>
       <property name="text" >
        <string>I</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fontUnderline" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>true</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="toolTip" >
        <string>underline</string>
       </property>
       <property name="text" >
        <string>U</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Align:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignLeft" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_left.svg</normaloff>:/data/text_left.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignHCenter" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>align horizontal center</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_center.svg</normaloff>:/data/text_center.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignRight" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>align right</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_right.svg</normaloff>:/data/text_right.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignTop" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>align top</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_top.svg</normaloff>:/data/text_top.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignVCenter" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>align vertical center</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_vcenter.svg</normaloff>:/data/text_vcenter.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alignBottom" >
       <property name="minimumSize" >
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>align bottom</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="mscore.qrc" >
         <normaloff>:/data/text_bottom.svg</normaloff>:/data/text_bottom.svg</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Awl::ColorLabel" name="color" >
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>text color</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Offset</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="topMargin" >
       <number>9</number>
      </property>
      <item rowspan="2" row="0" column="0" >
       <layout class="QFormLayout" name="formLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_9" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="xOffset" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>relX:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDoubleSpinBox" name="rxOffset" />
        </item>
       </layout>
      </item>
      <item rowspan="2" row="0" column="1" >
       <layout class="QFormLayout" name="formLayout_2" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="yOffset" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_13" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>relY:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QDoubleSpinBox" name="ryOffset" />
        </item>
       </layout>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_11" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Unit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QRadioButton" name="mmUnit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>MM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QRadioButton" name="spatiumUnit" >
        <property name="text" >
         <string>Space</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QGroupBox" name="frame" >
     <property name="title" >
      <string>Frame</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QRadioButton" name="circleButton" >
          <property name="toolTip" >
           <string>draw circled frame</string>
          </property>
          <property name="text" >
           <string>Circle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="boxButton" >
          <property name="toolTip" >
           <string>draw boxed frame</string>
          </property>
          <property name="text" >
           <string>Box</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QDoubleSpinBox" name="frameWidth" >
          <property name="toolTip" >
           <string>frame line width</string>
          </property>
          <property name="suffix" >
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Margin:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QDoubleSpinBox" name="paddingWidth" >
          <property name="toolTip" >
           <string>frame inner margin</string>
          </property>
          <property name="suffix" >
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Round:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QSpinBox" name="frameRound" />
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="Awl::ColorLabel" name="frameColor" >
          <property name="toolTip" >
           <string>frame color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="toolTip" >
      <string>align left</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Awl::ColorLabel</class>
   <extends>QFrame</extends>
   <header>awl/colorlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fontSelect</tabstop>
  <tabstop>fontSize</tabstop>
  <tabstop>fontBold</tabstop>
  <tabstop>fontItalic</tabstop>
  <tabstop>fontUnderline</tabstop>
  <tabstop>alignLeft</tabstop>
  <tabstop>alignHCenter</tabstop>
  <tabstop>alignRight</tabstop>
  <tabstop>alignTop</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>alignVCenter</tabstop>
  <tabstop>alignBottom</tabstop>
  <tabstop>xOffset</tabstop>
  <tabstop>rxOffset</tabstop>
  <tabstop>yOffset</tabstop>
  <tabstop>ryOffset</tabstop>
  <tabstop>mmUnit</tabstop>
  <tabstop>spatiumUnit</tabstop>
  <tabstop>frame</tabstop>
  <tabstop>circleButton</tabstop>
  <tabstop>boxButton</tabstop>
  <tabstop>frameWidth</tabstop>
  <tabstop>paddingWidth</tabstop>
  <tabstop>frameRound</tabstop>
 </tabstops>
 <resources>
  <include location="mscore.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
